app/views/main_template.scala.html

```diff
@@ -71,21 +71,13 @@
 }
 }
 
-@contentHeaderTextAsHtml = @{
```

Comment:
what is this used for reddy before you go just deleting it?

Reply:
This code is outdated and commit as part of https://jira.tools.tax.service.gov.uk/browse/VTCCA-1232